kpatch-build/kpatch-build

```diff
@@ -1164,7 +1167,12 @@ cd "$TEMPDIR/patch" || die
 if [[ -z "$OOT_MODULE" ]]; then
 	KPATCH_BUILD="$SRCDIR"
 else
```

Comment:
`$(dirname "$CONFIGFILE")` doesn't sound great, I think we'll be better off using `$SRCDIR` instead.

```suggestion
if [[ -z "$OOT_MODULE" || ! -e "/lib/modules/$ARCHVERSION/build" ]]; then
	KPATCH_BUILD="$SRCDIR"
else
	KPATCH_BUILD="KPATCH_BUILD="/lib/modules/$ARCHVERSION/build"
fi
```

Reply:
I tried `$SRCDIR` and it didn't work because if script is invoked from directory with OOT module, it does invoke `make -C $SRCDIR`, which does nothing, because `-C` must have **kernel** source directory as its argument.

Reply:
`SRCDIR` should contain the path to kernel source directory here. Here where it gets assigned: https://github.com/dynup/kpatch/blob/master/kpatch-build/kpatch-build#L675. And here we make sure `USERSRCDIR` is set when we use `OOT_MODULE`: https://github.com/dynup/kpatch/blob/master/kpatch-build/kpatch-build#L660.

Reply:
But this repo's README has the invocation example, and it states that SRCDIR is assigned module's source directory: https://github.com/dynup/kpatch/blob/61fc3f10776017f21249e2cbd4b0639cbc325d9e/README.md?plain=1#L588

Reply:
Yes, you are right. This semantic change is rather confusing. So are you relying on user-specified config then (through `--config` option)? Because if it is not specified that way it will be set to `/boot/config-xxx` (https://github.com/dynup/kpatch/blob/master/kpatch-build/kpatch-build#L738).

It seems to me that instead of changing `--source` semantics it'll be better to add another option for `OOT_MODULE` case that will define `OOT_MODULE_PATH` or something in that vein. We then'll be able to keep using `SRCDIR` normally

Reply:
Yes, I thought about that. Will be working on it next week and remove confusing semantics. Thanks for reviewing!

Reply:
Added `-p | --oot-module-src` option, allowing to specify module source and use `-s` in conventional way, only for kernel. Should I change the README of the repo too?